### PR TITLE
git-column: add page

### PR DESCRIPTION
**Pull request by bl-ue**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #3953

### CODE REVIEW

File: pages/common/git-column.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# git column
+
+> Display data in columns.
+> More information: <https://git-scm.com/docs/git-column>.
+
+- List files and folders in the current directory, formatted as columns:
```
Comment (sbrl) [inline pages/common/git-column.md:6]: Generally, we try to avoid demonstrating the functionality of other commands in a tldr page. Can we reword this to be more focused on the functionality of the current subcommand at all?
```suggestion
- Format the standard input as multiple columns:
```

Same goes for the other examples.

File: pages/common/git-column.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`ls | git column --mode=column`
```
Comment (sbrl) [inline pages/common/git-column.md:8]: Missing token syntax?
```suggestion
`ls | git column --mode={{column}}`
```

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page! I've left some comments below for you to review.

**Review by sbrl — APPROVED** [review]
Thanks, @bl-ue!

**Review by navarroaxel — APPROVED** [review]
